test/sdirk_integrators.jl

```diff
@@ -0,0 +1,25 @@
+using DelayDiffEq, DiffEqProblemLibrary, Base.Test
+
+prob = prob_dde_1delay(1.0)
+
+# ODE algorithms
+algs = [GenericImplicitEuler(), GenericTrapezoid(),
```

Comment:
Why not test any Rosenbrocks?

Reply:
There are additional problems with Rosenbrocks, they still do not work.

Reply:
Should Rosenbrocks get added now?